<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeRGBFilterForm</class>
 <widget class="QDialog" name="ChangeRGBFilterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменить RGB фильтр</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>169</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="newComponent">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>39</y>
     <width>125</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>149</x>
     <y>19</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Значение</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="component">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>105</width>
     <height>117</height>
    </rect>
   </property>
   <property name="title">
    <string>Компонент</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="rComponent">
      <property name="text">
       <string>Красный</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="gComponent">
      <property name="text">
       <string>Зелёный</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="bComponent">
      <property name="text">
       <string>Синий</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>newComponent</tabstop>
  <tabstop>rComponent</tabstop>
  <tabstop>gComponent</tabstop>
  <tabstop>bComponent</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
